src/spec/integration/export.spec.coffee

```diff
@@ -205,3 +206,36 @@ describe 'Export integration tests', ->
     .catch (err) -> done _.prettify(err)
     .finally ->
       tempDir.removeCallback()
+
+  it 'should export labels of lenum and set of lenum', (done) ->
```

Comment:
I think this test is really about exporting labels with a different **encoding**. We can make the test description more descriptive that that is what this test case is trying to achieve and not just export labels.